docker-compose.yml

```diff
@@ -118,4 +118,4 @@ networks:
         ipam:
             driver: default
             config:
-                - subnet: 172.29.0.0/16
\ No newline at end of file
+                - subnet: 172.29.0.0/16
```

Comment:
Looks like there is an empty line or space. Please double-check and remove if any